documentation/meta/contribution/good_first_and_help_wanted_issues.md

```diff
@@ -68,25 +68,92 @@ Descriptions should:
     knowledge, but if they at all do (for example, to explain why a non-obvious
     solution is requested) then it must be included
 
+### Additional boilerplate for "good first issues"
+
+In addition to our standard issue template, good first issues should _also_
+contain the following block of requirements:
+
+```md
+## Good first issue checks
+
+- Have I filled out the PR template correctly?
+  - Did I include testing instructions?
+- Does my PR pass linting? (Test either via precommit or by running `just lint`
+  manually)
+- Do all the tests still pass?
+  - If JavaScript changes, run `pnpm -r run test`
+  - If API changes, run ` just api/test`
+  - If catalog changes, run `just catalog/test`
+```
+
 ---
 
-## Reply Templates
-
-- Scenario: A "first time contributor" asks if they can work on an issue.
-
-  ```md
-  Hi @{user}, thank you for your interest in contributing to Openverse! I've
-  assigned this issue to you. If you have any questions, please leave them here.
-  Please check out our
-  [welcome](https://docs.openverse.org/general/contributing.html) and
-  [quickstart](https://docs.openverse.org/general/quickstart.html) documentation
-  pages for getting started with setting up your local environment.
-  ```
-
-- Scenario: A community pull request author did not use the pull request
-  template or failed to fill out all sections correctly.
-  ```md
-  Hi @{user}, could you update your pr description to use the
-  [pull request template](https://github.com/WordPress/openverse/blob/main/.github/PULL_REQUEST_TEMPLATE/pull_request_template.md)?
-  If you have any questions please let us know in the comments.
-  ```
+## Reply templates and behaviors
+
+These predefined responses ("predefs") provide consistient solutions to common
+situations.
+
+If you find yourself dealing with a recurring scenario that _isn't_ included in
+this list, please submit a pull request to add it here.
+
+### Scenario 1: Issue Request
+
+A "first time contributor" asks if they can work on an issue.
+
+#### Initial Response
+
+```md
+Hi `@user`, thank you for your interest in contributing to Openverse! I've
+assigned this issue to you. If you have any questions, you may leave them here.
+
+Please check out our
+[welcome](https://docs.openverse.org/general/contributing.html) and
+[quickstart](https://docs.openverse.org/general/quickstart.html) documentation
+pages for getting started with setting up your local environment.
```

Comment:
Might also be good to include a prompt for maintainers to share the particular "apps" quickstarts the contributor will need to keep in mind. Something like "Pay particular attention to the quickstarts linked for API, as that is where the changes need to be implemented, and you need to run the API locally to verify your changes".